src/main/java/org/quartz/classloading/CascadingClassLoadHelper.java

```diff
@@ -208,23 +244,25 @@ public InputStream getResourceAsStream(String name) {
       String relPath = pkgname.replace('.', '/').replace("%20", " ");
 
       // Get a File object for the package
-      URL resource = getResource(relPath);
-      if (resource == null) {
+      List<URL> resources = getResources(relPath);
+      if (resources.isEmpty()) {
```

Comment:
resources.size()>0 is preferred as I ran into issues with isEmpty many years ago. Can't remember why anymore though.

Reply:
or <1 in this case...